src/routes/SlideRoute.ts

```diff
@@ -0,0 +1,8 @@
+import express from 'express'
+import { updatePresentation } from '../controllers/SlideController'
+
+const router = express.Router()
+
+router.put('/:id', updatePresentation)
```

Comment:
Use named imports instead of destructing to avoid future naming conflicts and to keep our code consistent.

```import SlideController from '../controllers/SlideController'``` 

``` router.put('/:id', SlideController.updatePresentation)```